packages/cashier/src/stores/payment-agent-store.js

```diff
@@ -117,10 +117,10 @@ export default class PaymentAgentStore {
                     supported_banks: payment_agent?.supported_payment_methods || payment_agent?.supported_banks,
                     urls: payment_agent?.urls || payment_agent?.url,
                 });
-                if (payment_agent.supported_banks) {
-                    const supported_banks_array = payment_agent?.supported_payment_methods
-                        ? payment_agent.supported_payment_methods.map(bank => bank.payment_method)
-                        : payment_agent.supported_banks.split(',');
+                if (payment_agent.supported_payment_methods) {
+                    const supported_banks_array = payment_agent.supported_payment_methods.map(
+                        bank => bank.payment_method
+                    );
```

Comment:
@george-usynin-binary instead of
`if (payment_agent.supported_payment_methods) {
                    const supported_banks_array = payment_agent.supported_payment_methods.map(
                        bank => bank.payment_method
                    );`
Could you just have this:
`                    const supported_banks_array = payment_agent?.supported_payment_methods.map(
                        bank => bank.payment_method
                    );`

Reply:
sure